src/integrate/harmony/script.R

```diff
@@ -0,0 +1,37 @@
+library(reticulate)
+library(harmony)
+library(anndata)
```

Comment:
I might even be that we do not need this import, right?

Reply:
True. I'd prefer to keep the anndata import, as it allows to debug the anndata object a little more easily.